cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -1,1122 +0,0 @@
-            p("Lists all privileges for role 'deniedFunctionExecutor'")
-            resultTable()
-            p("The `apoc.any.property` and `apoc.any.properties` is blocked, as well as any other procedures starting with `apoc.any.prop`.")
-          }
-        }
-
-        section("The `EXECUTE BOOSTED USER DEFINED FUNCTION` privilege", "boosted-execute-function-subsection", "enterprise-edition") {
-          p(
-            """The ability to execute a user defined function (UDF) with elevated privileges can be granted via the `EXECUTE BOOSTED USER DEFINED FUNCTION` privilege.
-              |A user with this privilege is allowed to execute the UDFs matched by the <<name-globbing, name-globbing>>
-              |without the execution being restricted to their other privileges.
-              |There is no need to grant an individual `EXECUTE USER DEFINED FUNCTION` privilege for the functions either,
-              |as granting the `EXECUTE BOOSTED USER DEFINED FUNCTION` includes an implicit `EXECUTE USER DEFINED FUNCTION` grant for them.
-              |A denied `EXECUTE USER DEFINED FUNCTION` still denies executing the function.
-              |The following query shows an example of how to grant this privilege:""".stripMargin)
-          query("GRANT EXECUTE BOOSTED FUNCTION apoc.any.properties ON DBMS TO boostedFunctionExecutor", ResultAssertions(r => {
-            assertStats(r, systemUpdates = 1)
-          })) {
-            statsOnlyResultTable()
-            p(
-              """Users with the role 'boostedFunctionExecutor' can then run `apoc.any.properties` with full privileges,
-                |seeing every property on the node/relationship not just the properties that the user has `READ` privilege on.""".stripMargin)
-          }
-          p("The resulting role has privileges that only allow executing the UDF `apoc.any.properties`, but with elevated execution:")
-          query("SHOW ROLE boostedFunctionExecutor PRIVILEGES AS COMMANDS", NoAssertions) {
-            p("Lists all privileges for role 'boostedFunctionExecutor'")
-            resultTable()
-          }
-
-          note {
-            p("The `EXECUTE BOOSTED USER DEFINED FUNCTION` privileges do not apply to built-in functions, as they have no concept of elevated privileges.")
-          }
-
-          p(
-            """While granting `EXECUTE BOOSTED USER DEFINED FUNCTION` on its own allows the UDF to be both executed and given elevated privileges during the execution,
-              |the deny behaves slightly differently and only denies the elevation and not the execution. However, a user with only a granted `EXECUTE BOOSTED USER DEFINED FUNCTION`
-              |and a denied `EXECUTE BOOSTED USER DEFINED FUNCTION` denies the execution as well.
-              |This is the same behavior as for `EXECUTE BOOSTED PROCEDURE`, for examples see <<boosted-execute-procedure-subsection>>""".stripMargin)
-        }
```

Comment:
This don't seem to have been ported over?

Reply:
https://neo4j.com/docs/cypher-manual/4.3/access-control/dbms-administration/#access-control-execute-boosted-user-defined-function

Added to a card to fix this in another PR.

Reply:
As long as it gets fixed and doesn't entirely disappear